xarray/core/computation.py

```diff
@@ -1690,8 +1690,8 @@ def dot(
     dims: Dims = None,
     **kwargs: Any,
 ):
-    """Generalized dot product for xarray objects. Like np.einsum, but
-    provides a simpler interface based on array dimensions.
+    """Generalized dot product for xarray objects. Uses ``np.einsum``, but
```

Comment:
I preferred the existing version with `like` since it might not use the numpy function?